app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
For the time being i prefer to keep it simple and do a text-only email - can eliminate the html mailer template

Reply:
deleted